framework/aster-frame/src/task/task.rs

```diff
@@ -270,13 +271,15 @@ impl TaskOptions {
 
     /// Build a new task without running it immediately.
     pub fn build(self) -> Result<Arc<Task>> {
-        /// all task will entering this function
-        /// this function is mean to executing the task_fn in Task
-        extern "sysv64" fn kernel_task_entry() {
```

Comment:
Is it possible to replace `extern "sysv64"` with `extern "C"`? The latter should be available on all CPU architectures and is more light-weight than introducing a dedicated macro.

Reply:
It's possible, at least replacing all `sysv64` with `C` doesn't affect the syscall test and the regression test.

However, given that `extern "C"` is a more common choice, but `extern "sysv64"` is used here instead, I'm not sure if there is any special reason that makes it necessary.

Reply:
> Is it possible to replace `extern "sysv64"` with `extern "C"`? 

@junyang-zh What do you think?

Reply:
The initial adoption of "sysv64" is for the aster-frame entry point, where we write assembly to initialize function calling environment in x86. At that time we used "sysv64" just because some reference implementations used it for x86. I think `extern "C"` as a multi-arch ABI is suitable for this and we could use `extern "C"` for all non-boot related exported functions.

As for the entry points, that's defined in `aster_frame::arch::x86` and I think it is not such intolerable to leave it as is.